Documentation/install/azure/azure-private.md

```diff
@@ -0,0 +1,56 @@
+# Azure private clusters
+
+Use Tectonic Installer to create clusters within an existing private Azure network. In addition to the standard Azure requirements, private clusters also require:
+* An existing Virtual Network with a pre-configured and connected VPN link.
+* A DNS domain pre-registered into Azure DNS, or a 3rd party DNS server that supports DNS UPDATE operations.
+
+Once these requirements are met, follow the guide to [Install Tectonic on Azure with Terraform][azure-terraform].
+
+## Connected VPN link
+
+Private Azure networks require that an external VPN be available and running prior to running Tectonic Installer.
+
+For more information, see the Microsoft Azure document [Connect an on-premises network to Azure using a VPN gateway][connect-azure].
+
+## DNS domain or server
+
+Private clusters must use an external DNS provider, rather than Azure’s default DNS implementation.
+
+Tectonic will use DNS only, rather than Load Balancers, to route traffic to your private clusters on Azure. Boot a cluster that uses an API Fully Qualified Domain Name that points to the list of master private IP addresses.
+
+Follow the instructions to set up [DNS delegation and custom zones via Azure DNS][azure-dns].
+
+## Tectonic Installer
+
+When creating your cluster, set `tectonic_azure_private_cluster` to `true` to ensure that no public endpoints will be created as a result of the installation process.
+
+## Node Management
```

Comment:
*"Discovery"* instead of *"Management"* (both in the title and the phrase below it).